apps/vocabulary-operations/main.go

```diff
@@ -86,10 +86,11 @@ func main() {
 	intersectionPtr := flag.Bool("intersection", false, "generates the intersection of pb files")
 	differencePtr := flag.Bool("difference", false, "generates the difference of pb files")
 	exportPtr := flag.Bool("export", false, "export a given pb file as a csv file")
+	companyPtr := flag.Bool("company", false, "egenerates uniqueness within company")
```

Comment:
Please update this to "filterCommon" and make the description match.
I think the flag should be "--filter-common" to match Unix command line conventions.